app/models/ncr/work_order.rb

```diff
@@ -114,13 +114,23 @@ def approving_official
       self.approvers.first
     end
 
-    def current_approver_email_address
+    def current_approver
       if self.pending?
-        self.currently_awaiting_approvers.first.email_address
+        self.currently_awaiting_approvers.first
```

Comment:
change it all to not use self!? (haven't seen that style used in the rest of the project)